src/models/index.js

```diff
@@ -32,7 +32,8 @@ try {
   sequelize.authenticate();
   sequelize.sync();
 } catch (e) {
-  Logger.console.error(e);
+  const logger = new Logger('DatabaseConnectionError');
+  logger.error(e);
```

Comment:
Im not sure about that logger usage. Is it right?

Reply:
Yes it's a way better now
But I would initialize Logger outside of the catch statement